doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -712,9 +712,9 @@ And this maximum is:
 
```

Comment:
you're right, but if we use designation the root would be the result of its children so it's weird in any case... I don't know what is best...